src/main/java/org/sonar/squidbridge/annotations/ActivatedByDefault.java

```diff
@@ -24,6 +24,11 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
```

Comment:
Do we want to release the next version as 2.7 or 3.0?

Reply:
No idea. I'm gonna discuss the topic with PMs.

Reply:
And 2.7 it will be.